intellij/src/nl/tudelft/watchdog/intellij/logic/ui/listeners/GeneralActivityListener.java

```diff
@@ -10,7 +11,7 @@
  * A listener that determines whether there was general activity in the IntelliJ
  * window.
```

Comment:
Mention self-desctruction capabilities via dispose()?

Reply:
That is already deducible from the `implements Disposable`, so I would say this is not necessary. WDYT?

Reply:
I'd prefer to have this explicit.